### PR TITLE
feat(booking): add new fields to passenger data

### PR DESCRIPTION
**Pull request by matthewp5**
- add new loyalty programme fields to the OfferRequestPassenger interface
- validate new loyalty programme fields on order creation

### CODE REVIEW

File: src/booking/OfferRequests/OfferRequestsTypes.ts [hunk 1]
```diff
@@ -50,6 +75,21 @@ export interface OfferRequestPassenger {
...
+export interface LoyaltyProgrammeAccount {
+  /**
+   * The passenger's account number for this **Loyalty Programme Account**.
+   */
+  account_number: string
+
+  /**
+   * The IATA code for the airline that this **Loyalty Programme Account** belongs to.
+   */
+  airline_iata_code: string
```
Comment (igorp1) [inline src/booking/OfferRequests/OfferRequestsTypes.ts:90]: How about this?
```suggestion
  airline_iata_code: Airline['iata_code']
```

**Review by igorp1 — APPROVED** [review]
LGTM!

**Review by lapa182 — CHANGES_REQUESTED** [review]
@matthewp5 could create a unit test to this change, please?

File: src/booking/OfferRequests/OfferRequestsTypes.ts (hunk 1 above)
Comment (matthewp5) [reply src/booking/OfferRequests/OfferRequestsTypes.ts]: I like it, done!

**Review by lapa182 — APPROVED** [review]

**Comment by lapa182** [comment]
Thanks @matthewp5 !

**Pull request merged**